experimental/rust-tide/README.md

```diff
@@ -49,32 +49,28 @@ Templates are used to create your local project and start your development. When
         }
     ```
 
-1. Your application will be rebuild and republished so refresh http://localhost:8000/ it will now say `Hello, Tide`
+1. Your application will rebuild and republish so refresh http://localhost:8000/ it will now say `Hello, Tide`
 
 
 ## Debugging
 
-To debug your application running in a container, start the container using:
+This stack is configured to run a remote gdb server in the app container. 
+You can connect to the gdb server on localhost:1234 with the tools of your choice.
+However the stack also comes preconfigured to integrate with VSCode and you can debug the application with the following steps:
 
-```bash
-    appsody debug --docker-options "--cap-add=SYS_PTRACE --security-opt seccomp=unconfined"
-```
-
-The command will start the LLDB platform and wait for incoming connections from any address to port 1234. 
-
-You can connect `lldb` in remote debug mode to this container as follows:
-
-```bash
-lldb \
-  -o "platform select remote-linux" \
-  -o "platform connect connect://localhost:1234" \
-  -o "platform settings -w /project/server/bin/target/debug" \
-  -o "file rust-tide-server" 
-```
+1. Start the container in debug mode
+   This is done by pressing `CTRL + SHFT + B` and selecting `Appsody: debug` 
+   Or through the menu option `Terminal --> Run Task --> Appsody: debug`
+   The command will start the the app with the gdb server and wait for incoming connections from any address to port 1234. 
 
-Once in lldb, you can use `breakpoint set` to set breakpoints in your application, and then `run` to start the app.
+2. Run the Appsody copy task to get the symbols for the the local debugger.
+   This is done by pressing `CTRL + SHFT + B` and selecting `Appsody: copy` 
+   Or through the menu option `Terminal --> Run Task --> Appsody: copy`
+    
+3. Now we can debug the program with pressing `F5`
```

Comment:
I get this error at this point:
```
Unable to start debugging. Launch options string provided by the project system is invalid. Unable to determine path to debugger.  Please specify the "MIDebuggerPath" option.
```
Have you seen this before?

Reply:
That looks like you don't have gdb installed I have put notes in the README to point towards how to do this for Linux, Mac and Windows.

Reply:
Have you tried using `CodeLLDB`? We had some issues with it where the process was still running in the container after the debug session in Vscode was finished. Did you encounter the same issue, and is this the solution to get around it? If so it might be good to update the other Rust stack to be consistent

Reply:
Hey @EnriqueL8 
Yes I recreated the issue with `CodeLLDB` 
I also looked at using lldb connected directly to ensure the VSCode wasn't causing an issue. 
Despite the simpler approach the issues continued along with a general inability to step through the process execution and some random crashes.
I don't think LLDB remote debugging is up to it at the moment hence taking the gdbserver approach.
Also note that the gdbserver approach is not perfect but it is a lot more robust and usable than lldb remote.
FYI I also evaluated installing https://www.gdbgui.com/ into the container.
This again this wasn't perfect but it did remove the need to integrate with external developer tools but meant the developer would need to use a none familiar tool.
I think the implementation in this PR gives the most flexibility but I'll always open to reconsidering it if users have different preferences.

Reply:
Nice, thanks for the info! Might be good documenting this somewhere